frame/staking-rewards/src/test/test_reward_accumulation_hook.rs

```diff
@@ -505,7 +505,7 @@ fn test_accumulate_rewards_hook() {
 			]);
 
 			check_events([crate::Event::<Test>::MaxRewardsAccumulated {
-				pool_id: bobs_pool_id,
+				pool_id: C::ID,
```

Comment:
was this change required? (curious why there were changes in `check_events` and not in `check_rewards`)

Reply:
It's because we no longer require an explicit `pool_id` to verify rewards, staked asset ID(`pool_asset_id`) is already the pool id. Made a change to get rid of it in the test anyway